subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -72,6 +72,13 @@ struct hci_driver_data {
 	bt_hci_recv_t recv;
 };
 
+#if !defined(CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE)
+static struct node_rx_hci_evt {
+	struct node_rx_hdr hdr;
+	struct net_buf *buf;
+} hci_evt;
+#endif /* !CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE */
```

Comment:
Do we need to guard this struct? 

If not guarded, you should be able to use IS_ENABLED and the linker should be able to remove the struct if not used